diff --git a/SabreTools/SabreTools.csproj b/SabreTools/SabreTools.csproj
index 9adc4c7a..cb15305b 100644
--- a/SabreTools/SabreTools.csproj
+++ b/SabreTools/SabreTools.csproj
@@ -10,6 +10,7 @@
     <LangVersion>latest</LangVersion>
     <Nullable>enable</Nullable>
     <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
+    <TargetName>sabretools</TargetName>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <Version>1.1.2</Version>
 
